cvat/apps/dataset_manager/bindings.py

```diff
@@ -1108,24 +1252,25 @@ def map_label(name, parent=''): return label_cat.find(name, parent)[0]
         return convert_cvat_anno_to_dm(cvat_frame_anno, label_attrs, map_label)
 
 
-class CvatTaskDataExtractor(dm.SourceExtractor, CVATDataExtractorMixin):
-    def __init__(self, task_data, include_images=False, format_type=None, dimension=DimensionType.DIM_2D):
+class CvatTaskOrJobDataExtractor(dm.SourceExtractor, CVATDataExtractorMixin):
+    def __init__(self, instance_data: Union[TaskData, JobData], include_images=False, format_type=None, dimension=DimensionType.DIM_2D):
```

Comment:
Wouldn't it make sense to use `CommonData` as an annotation here? I assume that was the whole purpose of the class, to serve as a common interface.

Same in many other cases below where `TaskData, JobData` or `JobData, TaskData` is used.